samples/tests/Console.js

```diff
@@ -30,3 +54,20 @@ console.log(longstr);
 console.log("\nTesting a string too long...");
 longstr += '6';  // length 256
 console.log(longstr);
+
+console.log("\nTesting console.assert... ");
+console.assert(true);
+console.log("expected result: true");
+console.assert(false, "My lover!");
```

Comment:
O_o let's make the sample string less interesting? :)

This should be throwing an error - with a new patch from @jprestwo it will be AssertionError like Node docs say. So you should do a try/catch.